### PR TITLE
Document wxStdDialogButtonSizer::GetXXXButton()

### PR DESCRIPTION
**Pull request by wh11204**
See issue #24863

### CODE REVIEW

File: interface/wx/sizer.h [hunk 1]
```diff
@@ -1041,6 +1041,41 @@ class wxStdDialogButtonSizer : public wxBoxSizer
     */
     void Realize();
 
+    /**
+        Returns the affirmative button for the sizer.
+
+        Affirmative buttons have ID awxID_OK, wxID_YES or wxID_SAVE
```
Comment (PBfordev) [inline interface/wx/sizer.h:1047]:
```suggestion
        Affirmative buttons have ID @c wxID_OK, @c wxID_YES or @c wxID_SAVE
```

More importantly, are the listed IDs really the only allowed IDs, as one can add any button and set any button as one of the known buttons?

File: interface/wx/sizer.h [hunk 2]
```diff
@@ -1041,6 +1041,41 @@ class wxStdDialogButtonSizer : public wxBoxSizer
...
+        Returns the negative button for the sizer.
+
+        Negative buttons have ID wxID_NO
+    */
+    wxButton* GetNegativeButton() const;
+
+    /**
+        Returns the cancel button for the sizer.
+
+        Cancel buttons have ID wxID_CANCEL or wxxID_CLOSE
```
Comment (PBfordev) [inline interface/wx/sizer.h:1068]:
```suggestion
        Cancel buttons have ID @c wxID_CANCEL or @c wxID_CLOSE
```

**Review by PBfordev — COMMENTED** [review]
Just fixing two typos (and proposing to use `@c` for constants) and asking one question applying to several methods.

BTW, I would end all the sentences with a period, not just the first one.

**Comment by vadz** [comment]
Thanks for documenting them, will apply soon.

**Pull request closed without merging**